apps/web-client/src/lib/components/Tables/InventoryTables/StockBookRow.svelte

```diff
@@ -51,6 +50,9 @@
 	<td data-property="editedBy" class="show-col-xl table-cell-max">
 		{editedBy}
 	</td>
+	<td data-property="category" class="show-col-xl table-cell-max">
```

Comment:
I think lines 53-55 (the new category cell) should be moved after the BookOutPrintCell.
This is the current stock view:
![image](https://github.com/librocco/librocco/assets/396425/eb26ce01-d20c-4926-8bc6-462bd25fe3e1)